app/src/main/java/com/onegini/mobile/exampleapp/network/client/SecureResourceClient.java

```diff
@@ -17,75 +17,34 @@
 
 public class SecureResourceClient {
 
-  private static final GsonConverter gsonConverter = new GsonConverter(new GsonBuilder().disableHtmlEscaping().create());
-
   private SecureResourceClient() {
   }
 
-  /*
-   * user clients
-   */
-
-  // preparing the client using Retrofit 1.9
   public static <T> T prepareSecuredUserRetrofitClient(final Class<T> clazz, final Context context) {
     final OkHttpClient okHttpClient = OneginiSDK.getOneginiClient(context).getUserClient().getResourceOkHttpClient();
     return prepareSecuredRetrofitClient(clazz, context, okHttpClient);
   }
 
-  // preparing the client using Retrofit 2.X
-  public static <T> T prepareSecuredUserRetrofit2Client(final Class<T> clazz, final Context context) {
-    final OkHttpClient okHttpClient = OneginiSDK.getOneginiClient(context).getUserClient().getResourceOkHttpClient();
-    return prepareSecuredRetrofit2Client(clazz, context, okHttpClient);
-  }
-
-  /*
-   * implicit user clients
-   */
-
-  // preparing the client using Retrofit 1.9
   public static <T> T prepareSecuredImplicitUserRetrofitClient(final Class<T> clazz, final Context context) {
     final OkHttpClient okHttpClient = OneginiSDK.getOneginiClient(context).getUserClient().getImplicitResourceOkHttpClient();
     return prepareSecuredRetrofitClient(clazz, context, okHttpClient);
   }
 
-  // preparing the client using Retrofit 2.X
-  public static <T> T prepareSecuredImplicitUserRetrofit2Client(final Class<T> clazz, final Context context) {
-    final OkHttpClient okHttpClient = OneginiSDK.getOneginiClient(context).getUserClient().getImplicitResourceOkHttpClient();
-    return prepareSecuredRetrofit2Client(clazz, context, okHttpClient);
-  }
-
-  /*
-   * anonymous clients
-   */
-
-  // preparing the client using Retrofit 1.9
   public static <T> T prepareSecuredAnonymousRetrofitClient(final Class<T> clazz, final Context context) {
     final OkHttpClient okHttpClient = OneginiSDK.getOneginiClient(context).getDeviceClient().getAnonymousResourceOkHttpClient();
     return prepareSecuredRetrofitClient(clazz, context, okHttpClient);
   }
 
-  // preparing the client using Retrofit 2.X
-  public static <T> T prepareSecuredAnonymousRetrofit2Client(final Class<T> clazz, final Context context) {
-    final OkHttpClient okHttpClient = OneginiSDK.getOneginiClient(context).getDeviceClient().getAnonymousResourceOkHttpClient();
-    return prepareSecuredRetrofit2Client(clazz, context, okHttpClient);
-  }
-
-
-  // preparing the client using Retrofit 2.X
-  private static <T> T prepareSecuredRetrofit2Client(final Class<T> clazz, final Context context, final OkHttpClient okHttpClient) {
+  private static <T> T prepareSecuredRetrofitClient(final Class<T> clazz, final Context context, final OkHttpClient okHttpClient) {
     final OneginiClient oneginiClient = OneginiSDK.getOneginiClient(context);
     final Retrofit retrofit = new Retrofit.Builder()
         .client(okHttpClient)
```

Comment:
We should add "/" to original OneginiConfigModel.resourceBaseURL and remove it here